src/__tests__/renderer-test.js

```diff
@@ -10,36 +10,62 @@ function getNodes(text) {
   return reparsed.document.nodes;
 }
 
+// Asserts whether the original Markdown string is the same as the one created
+// by parsing and re-rendering the original.
+function assertSymmetry(text, value) {
```

Comment:
This seems like it functionality performs the same job as `getNodes`?

Reply:
Agreed, I think the snapshot approach is preferable as it will make any regressions or bugs easier to identify and fix. It also makes changes to the renderer output trackable in the git history.